ca_gazebo/models/camera/model.sdf

```diff
@@ -0,0 +1,35 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='camera'>
+    <static>true</static>
+    <pose>-1 0 2 0 1 0</pose>
+    <link name='link'>
+      <visual name='visual'>
+        <geometry>
+        <mesh>
+          <uri>model://camera/meshes/camera.stl</uri>
+          <scale>2.5 2.5 2.5</scale>
+        </mesh>
+        </geometry>
+      </visual>
+      <sensor name='my_camera' type='camera'>
+        <camera>
+          <save enabled="true">
```

Comment:
I don't recommend you to save the frames. That's not necessary.

Reply:
Adressed